pkg/provider/provider.go

```diff
@@ -23,7 +25,15 @@ type Provider interface {
 	Decommision(node *v1.Node) error
 }
 
+type Providers struct {
+	AvailableProviders []string
+	Scaffolds          map[string]string
+}
+
+var providers = Providers{}
+
 func Bootstrap(manifestPath string, force bool) (Provider, error) {
+	GetAvailableProviders()
```

Comment:
considering how this function basically is just called for side-effects, I definitely think it should definitely be made private